docs/data.rst

```diff
@@ -210,3 +210,38 @@ you might need to modify the DB path, or the dataset name.
 We provide three `group` datasets since we often use it to test federation of three test
 servers.
 
+Importing sample FASTA:
+
+- Download http://hgdownload.cse.ucsc.edu/goldenpath/hg19/bigZips/hg19.fa.gz
+- :code:`gunzip hg19.fa.gz` to unzip
+- Import by running :code:`candig_repo add-referenceset candig-example-data/registry.db /path/to/hg19a.fa -d "hg19a reference genome" --name hg19a`
+
+
+Import sample VCF:
+
+- To work with a certain `group`, download the `tar` file and load script.
+- Assuming we are dealing with `group1` from release v1.0.0, download
+    - https://github.com/CanDIG/test_data/releases/download/v1.0.0/group1.tar
+    - https://github.com/CanDIG/test_data/releases/download/v1.0.0/group1_load.sh
+    - https://github.com/CanDIG/test_data/releases/download/v1.0.0/group1_clinphen.json
+- :code:`tar xvf group1.tar` to be run for unarchiving
+- In `group1_load.sh`
+    - rename all instances of :code:`GRCh37-lite` to :code:`hg19a`
```

Comment:
this may not be always the case, as different places use different referenceSet, which may be GRCh37-lite (at GSC or UHN), or hg19 (if downloaded from above), or something else.

Reply:
Let me re-write this line then

Reply:
Sorry, what did you change it to? i might have missed the changes..

Reply:
I had added a warning earlier but added that to the line in question as well. Does this work or should I make more changes?

Reply:
Oh sorry, I totally missed the changes.